CONTRIBUTING.rst

```diff
@@ -32,6 +39,10 @@ covering all supported Python and Django version with::
 
     tox
 
+With docker run tests::
+
+    docker-compose run --rm cli bash -c "pip install -r requirements_dev.txt; make test"
```

Comment:
Is this really easier than running `pip install --user tox; tox`? If Docker supports one-shot services maybe we should have one to run the tests?

Reply:
that is always a matter of personal flavour. I like to prevent messing around locally with dozens of different virtualenvs. but I'm not the maintainer. I offered what I ran locally and others asked for it. so its up to you to find the right way for this lib.